app/controllers/my_module_repository_snapshots_controller.rb

```diff
@@ -86,12 +87,21 @@ def select
                 .update(selected: nil)
       repository_snapshot.update!(selected: true)
     end
+    @my_module.update_report_references(params[:repository_id] || params[:repository_snapshot_id])
```

Comment:
Passing unchecked params directly to update method?

Reply:
Not sure what else should I check here?